From 114ab3037de3b0f9b35cf023b64c8a9b76070065 Mon Sep 17 00:00:00 2001
From: Debian GnuPG Maintainers <pkg-gnupg-maint@lists.alioth.debian.org>
Date: Tue, 14 Apr 2015 10:02:31 -0400
Subject: [PATCH 6/7] avoid beta warning

avoid self-describing as a beta

Using autoreconf against the source as distributed in tarball form
invariably results in a package that thinks it's a "beta" package,
which produces the "THIS IS A DEVELOPMENT VERSION" warning string.

since we use dh_autoreconf, i need this patch to avoid producing
builds that announce themselves as DEVELOPMENT VERSIONs.

See discussion at:

 http://lists.gnupg.org/pipermail/gnupg-devel/2014-November/029065.html
---
 autogen.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index b23855061..9b86d3ff9 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -229,24 +229,24 @@ if [ "$myhost" = "find-version" ]; then
     esac
 
     beta=no
-    if [ -e .git ]; then
+    if false; then
       ingit=yes
       tmp=$(git describe --match "${matchstr1}" --long 2>/dev/null)
       tmp=$(echo "$tmp" | sed s/^"$package"//)
       if [ -n "$tmp" ]; then
           tmp=$(echo "$tmp" | sed s/^"$package"//  \
                 | awk -F- '$3!=0 && $3 !~ /^beta/ {print"-beta"$3}')
       else
           tmp=$(git describe --match "${matchstr2}" --long 2>/dev/null \
                 | awk -F- '$4!=0{print"-beta"$4}')
       fi
       [ -n "$tmp" ] && beta=yes
       rev=$(git rev-parse --short HEAD | tr -d '\n\r')
       rvd=$((0x$(echo ${rev} | dd bs=1 count=4 2>/dev/null)))
     else
       ingit=no
-      beta=yes
-      tmp="-unknown"
+      beta=no
+      tmp=""
       rev="0000000"
       rvd="0"
     fi
-- 
2.27.0

